arangod/Aql/grammar.h

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.7.  */
+/* A Bison parser, made by GNU Bison 3.5.1.  */
```

Comment:
Any reason to go back to this version?

Reply:
It's a generated file, and whatever is in there will likely get overwritten soon by a new build. So I wouldn't worry about it.

Reply:
Anyway, reverted all these changes in this PR.